packages/autorest.gotest/test/integrationtest/output/compute/armcompute/sample_live_test.go

```diff
@@ -39,6 +40,7 @@ func (testsuite *SampleTestSuite) SetupSuite() {
 	testsuite.cred, testsuite.options = testutil.GetCredAndClientOptions(testsuite.T())
 	testsuite.fakeStepVar = "signalrswaggertest4"
 	testsuite.resourceName = "signalrswaggertest4"
+	testsuite.testPrefix = testutil.GenerateAlphaNumericID(testsuite.T(), "test", 6)
```

Comment:
wondering does this function exist in the testutil?

Reply:
I've already released the latest version with the feature.